packages/codeeditor/src/editor.ts

```diff
@@ -451,10 +451,18 @@ namespace CodeEditor {
 
     /**
      * Repaint the editor.
+     * 
+     * #### Notes
+     * A repainted editor should fit to its host node.
      */
     refresh(): void;
 
     /**
+     * Resize the editor to fit its host node.
+     */
+    resizeToFit(): void;
```

Comment:
`refresh` (to me) doesn't necessarily denote resizing. Do you think it's possible for an editor to do *less* work if it knows it's just a potential resize?